crates/topos/src/components/sequencer/commands/run.rs

```diff
@@ -19,6 +19,12 @@ pub struct Run {
     )]
     pub subnet_jsonrpc_endpoint: String,
 
+    // Optional explicit websocket endpoint for the subnet jsonrpc api. If this parameter is not provided,
+    // it will be derived from the `subnet_jsonrpc_endpoint`.
+    // Full uri value is expected, e.g. `wss://arbitrum.infura.com/v3/ws/mykey` or `ws://127.0.0.1/ws`
+    #[clap(long, env = "SUBNET_WEBSOCKET_ENDPOINT")]
+    pub subnet_websocket_endpoint: Option<String>,
```

Comment:
I'd mention about that on the description of `subnet_jsonrpc_endpoint` too

(also `///` for the rustdoc)